nixos/modules/programs/gpu-screen-recorder.nix

```diff
@@ -34,6 +34,8 @@ in {
       capabilities = "cap_sys_nice+ep";
```

Comment:
Can you remove this cap sys nice section? it has to be removed at the moment because it breaks desktop portal capture (pipewire). It's mentioned here: https://git.dec05eba.com/gpu-screen-recorder/tree/extra/meson_post_install.sh#n7